paperqa/clients/utils.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
```

Comment:
Can we somehow remove this module? We already have `utils.py` one level up, can we just use that one?

Reply:
yea, i wanted this to truly be a submodule, but I suppose maybe we'd reuse this.